apps/prairielearn/python/zygote.py

```diff
@@ -115,7 +115,6 @@ def find_spec(
             for module in self.forbidden_modules
         ):
             raise ImportError(f'module "{fullname}" is not allowed.')
-        return None
```

Comment:
If the return is removed, could a type annotation be added for returning none?

Reply:
Actually, I remember the `return None` actually signifying something here. Changed in 765ad64f7.